### DISCUSSION TITLE
SQLAlchemy Composite Column Type Not working

### DISCUSSION THREAD

**Discussion by erenz14** [fastapi/sqlmodel#564] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from sqlmodel import SQLModel, Field
from sqlalchemy.orm import composite
from typing import Optional

class Point(object):
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def __composite_values__(self):
        return self.x, self.y

    def __repr__(self):
        return f"Point(x={self.x!r}, y={self.y!r})"

    def __eq__(self, other):
        return isinstance(other, Point) and other.x == self.x and other.y == self.y

    def __ne__(self, other):
        return not self.__eq__(other)

class Vertex(SQLModel, table=True):

    id = Optional[int] = Field(default=None, primary_key=True)
    x1: int = Field(default=None)
    y1: int = Field(default=None)
    x2: int = Field(default=None)
    y2: int = Field(default=None)

    start = composite(Point, x1, y1)
    end = composite(Point, x2, y2)
```

### Description

SQLAlchemy describes the Composite Column Type here: https://docs.sqlalchemy.org/en/14/orm/composites.html

I'm trying to use Composite Column Type with SQLModel in a similar way as show above in the example, but I receive the following error:  `TypeError: cannot pickle 'property' object`

Has anybody successfully used the composite column type with SQLModel yet?

### Operating System

Windows

### SQLModel Version

0.0.8

### Python Version

3.8.0

**Comment by jpmckinney** [comment]
I have a similar question, in that SQLModel doesn't seem to support multicolumn indexes? I suppose I need to drop down to SQLAlchemy or Alembic to add a multicolumn index.
